docs/configuration/git-providers/bitbucket-enterprise.md

```diff
@@ -1,6 +1,11 @@
 # Bitbucket Stash
 
 Set your configuration values for your Bitbucket instance on the `values.yaml` file.
+
+NOTE: Since bitbucket uses OAuth1, you'll need to create a key pair to sign and validate the requests between codacy and the bitbucket enterprise instance.
+You can do it by running the command `ssh-keygen -t rsa -f mykey`
+Then, copy the private key from the `mykey` file and set it in the `consumerPrivateKey` and copy the public key from the `mykey.pub` file and set it in the `consumerPrivateKey`. 
```

Comment:
Please consider the suggestions below, namely:

* We should probably instruct the user not to use a passphrase for the key pair, correct?
* There was a typo in `consumerPrivateKey`, it should mention the `consumerPublicKey`.

```
**NOTE:**\
Since Bitbucket uses OAuth1, you'll need to create a key pair to sign and validate the requests between Codacy and the Bitbucket Enterprise instance:

1. Create the key pair by running the command below, making sure that you don't define a passphrase:

    `ssh-keygen -t rsa -f mykey`

1. Copy the private key from the file `mykey` and set it as the value of `consumerPrivateKey`.
1. Copy the public key from the file `mykey.pub` and set it as the value of `consumerPublicKey`.
```